drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -151,6 +148,7 @@ def items_without_answers():
             for item in items:
                 del item['feedback']
                 del item['zone']
+                item['displayName'] = self.ugettext(item['displayName'])
```

Comment:
@asadiqbal08 

> We have some initial default values in default_data.py. I think We can translate these initial static data. [...]

Since these defaults live in a separate module (i.e., they are not part of the `DragAndDropBlock` class), I don't think you can translate them using the new `ugettext` mechanism from `XBlock`. Also, zone titles can (and will :)) be modified by course authors; as soon as that happens, translations for default values won't apply anymore (even if an author only uses the default zones that are created when adding a new DnDv2 block to a unit, and doesn't add any additional zones).